### PR TITLE
fix(agw): Fixed memory leaks during execution of s1ap sanity test suite

### PR DESCRIPTION
**Pull request by rsarwad**
fix(agw): Fixes the issue, Fixes the issue, https://github.com/magma/magma/issues/6210

## Summary
All the memory leaks reported in issue, https://github.com/magma/magma/issues/6210 are not observed on latest master codebase. And provided fixes for which memory leaks observed on latest master codebase.

## Test Plan
Executed s1ap sanity test suite

### CODE REVIEW

File: lte/gateway/c/core/oai/tasks/nas/emm/TrackingAreaUpdate.c [hunk 1]
```diff
@@ -682,11 +682,8 @@ static int emm_tracking_area_update_accept(nas_emm_tau_proc_t* const tau_proc) {
       emm_sap.u.emm_as.u.establish.emergency_number_list = NULL;
 
       emm_sap.u.emm_as.u.establish.eps_network_feature_support =
-          calloc(1, sizeof(eps_network_feature_support_t));
-      emm_sap.u.emm_as.u.establish.eps_network_feature_support->b1 =
-          _emm_data.conf.eps_network_feature_support[0];
-      emm_sap.u.emm_as.u.establish.eps_network_feature_support->b2 =
-          _emm_data.conf.eps_network_feature_support[1];
+          (eps_network_feature_support_t*) &_emm_data.conf
```
Comment (ardzoht) [inline lte/gateway/c/core/oai/tasks/nas/emm/TrackingAreaUpdate.c:685]: could this lead to eps_network_feature_support ptr being freed later and then another place trying to access it? I assume we do a nullptr check if that's the case, just want to make sure this doesn't lead to crash

**Review by ardzoht — APPROVED** [review]
lgtm, thanks! just left one question regarding the `eps_network_feature_support` ptr

File: lte/gateway/c/core/oai/tasks/nas/emm/TrackingAreaUpdate.c (hunk 1 above)
Comment (rsarwad) [reply lte/gateway/c/core/oai/tasks/nas/emm/TrackingAreaUpdate.c:685]: Here we are not actually allocating memory, just sending address of parameter, eps_network_feature_support which is part of configuration structure.
While accessing we have null check.

**Review by ulaskozat — APPROVED** [review]
LGTM

**Comment by themarwhal** [comment]
Hey @ulaskozat & @rsarwad, there are too many conflicts to backport into 1.5. I'll be removing the apply-v1.5 label. Please raise a PR against the 1.5 branch. Thanks! :)

**Comment by themarwhal** [comment]
Actually, looks like there's a merge conflict for 1.6 as well. Would be great if they can be backported as a PR too. Thanks!

**Comment by ulaskozat** [comment]
cc @rsarwad for backporting PRs

**Comment by rsarwad** [comment]
> cc @rsarwad for backporting PRs

Two memory leaks were introduced by snippet of code that is neither in v1.5 nor in v1.6. So I shall check the memory in individual branches and update.

**Comment by ulaskozat** [comment]
but TAU one is there as far as I see.

**Pull request merged**